nbconvert/nbconvertapp.py

```diff
@@ -17,8 +17,9 @@
 from jupyter_core.application import JupyterApp, base_aliases, base_flags
 from traitlets.config import catch_config_error, Configurable
 from traitlets import (
-    Unicode, List, Instance, DottedObjectName, Type, CaselessStrEnum, Bool,
+    Unicode, List, Instance, DottedObjectName, Type, CaselessStrEnum, Bool, Undefined, Enum
```

Comment:
Ah right of course! 



--  
John Bohannon, PhD
Contributing Correspondent, Science Magazine
johnbohannon.org <http://www.johnbohannon.org/> 

> On Dec 14, 2015, at 8:57 AM, Matthias Bussonnier <notifications@github.com> wrote:
> 
> In nbconvert/nbconvertapp.py <https://github.com/jupyter/nbconvert/pull/141#discussion_r47523409>:
> 
> > @@ -17,8 +17,9 @@
> >  from jupyter_core.application import JupyterApp, base_aliases, base_flags
> >  from traitlets.config import catch_config_error, Configurable
> >  from traitlets import (
> > -    Unicode, List, Instance, DottedObjectName, Type, CaselessStrEnum, Bool,
> > +    Unicode, List, Instance, DottedObjectName, Type, CaselessStrEnum, Bool, Undefined, Enum
> @gonzolabs <https://github.com/gonzolabs> Type is already imported here :-P
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/jupyter/nbconvert/pull/141/files#r47523409>.
>